Python/Product/PythonTools/PythonTools/Logging/InMemoryLogger.cs

```diff
@@ -119,6 +131,18 @@ class InMemoryLogger : IPythonToolsLogger {
                 }
             }
 
+            lock (_analysisTiming) {
+                if (_analysisTiming.Any()) {
+                    res.AppendLine("Analysis timing:");
+                    foreach (var kv in _analysisTiming.OrderBy(kv => kv.Key)) {
+                        res.AppendFormat("    {0} (count {1}, {2} timeouts, max {3:N0}ms, mean {4:N2}ms)",
```

Comment:
Arguably the average isn't all that interesting here anyway (mean or otherwise), but I'm not going to write code for bucketing just for this display.